tsconfig.browser.json

```diff
@@ -0,0 +1,24 @@
+{
+	"compilerOptions": {
```

Comment:
Would be nice if we could use `"extends": "./tsconfig.json",` so that the changes needed just for the browser are more obvious.